cpp/server/handler.cc

```diff
@@ -294,6 +306,8 @@ void HttpHandler::GetEntries(evhttp_request* req) const {
                      "Missing or invalid \"end\" parameter.");
   }
 
+  const bool include_scts(GetBoolParam(query, "include_scts"));
+
```

Comment:
Maybe put that right next to line 321? And lift the comment too?

https://github.com/google/certificate-transparency/blob/master/cpp/server/handler.cc#L193